### ISSUE TITLE
Out of place back tick (accent grave)

### ISSUE THREAD

**Issue by wclodius2** [googlefonts/noto-fonts#2176]
## Defect Report
> Noto fonts can be obtained from https://github.com/googlefonts/noto-fonts/tree/master/unhinted/otf/ (unhinted OTF) or https://github.com/googlefonts/noto-fonts/tree/master/hinted/ttf/ (hinted TTF). If you find a problem with either of these then please use this template to file a defect report. For feature requests and other matters, you can use part of the template and delete what you don't need.

### Title 
 Out of place back tick (accent grave) when bracketed by apostrophes.

### Font 
Noto Serif

### Where the font came from, and when
https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,400,400italic,600,600italic%7CNoto+Serif:400,400italic,700,700italic%7CDroid+Sans+Mono:400,700)

### Font Version
Could not discover

### OS name and version
Mac OS Monterey version 12.0.1

### Application name and version
 Safari Version 15.1, Firefox Browser 94.0.1, Google Chrome Version 96.0.4664.55

### Issue
I am trying to display a back tick bracketed by two apostrophes, '`'. I am generating the text using Asciidoctor, which generates html using the font Noto Serif with the text '`'. I am getting either two apostrophes followed by a back tick, or a double quote followed by a back tick. I can't tell which. It is definitely not the expected '`'.

### Character data
The input characters are '`' or U+0027 U+0060 U+0027

### Screenshot
The problem is so well localized I find a screenshot to be overkill.

## Tools for reporting bugs
  Useful tools for reporting bugs are available at: https://github.com/googlei18n/
  
### Harfbuzz hb-view and hb-shape
  These are part of the HarfBuzz distribution and can help isolate if an issue is in the app/OS, shaping engine, or font.
  * hb-view renders the text with the exact font (for example, to see how ligatured characters shape) using your installed version of HarfBuzz.

For example:
```
  hb-view --font-file {path to font} --text-file {path to text file} --output-file '{sample}.png'
```

* hb-shape shows glyph selection and positioning

### Fontview
  * Fontview displays the text.

### Fontdiff
  * Fontdiff displays the text using two versions of the font side by side.

**Comment by wolfgang42** [comment]
I can reproduce this issue with MacOS 11.6.1 in Safari 15.1
(16612.2.9.1.30, 16612) using this URL:
https://fonts.google.com/?query=noto+serif&preview.text=%27%60%27&preview.text_type=custom

![image](https://user-images.githubusercontent.com/2261204/145334379-d6879284-c4ba-49a4-8990-bf47244581cf.png)


Interestingly, I *don't* see the issue in Chrome 96.0.4664.55 on the
same machine; instead the backtick shows up as tofu in this font:


Some experimentation leads me to believe that the backtick glyph is
being treated as though it has zero width; for example, `\`x` renders
as though it's an `x` with a grave accent over it, and `\`\`\`\`` just
shows up as a single backtick.

**Comment by mhsdesign** [comment]
The fonts hosted on google fonts seem to be of an older version: https://github.com/googlefonts/noto-fonts/issues/736#issuecomment-768889258

**Comment by simoncozens** [comment]
Agreed - the width issue now fixed in the released fonts in the repo, but it looks like we do need to onboard a new update to the version on Google Fonts.
